runtime/lua/vim/diagnostic.lua

```diff
@@ -890,22 +872,33 @@ M.handlers.signs = {
     end
 
     local ns = M.get_namespace(namespace)
-    if not ns.user_data.sign_group then
-      ns.user_data.sign_group = string.format('vim.diagnostic.%s', ns.name)
+    if not ns.user_data.sign_ns then
+      ns.user_data.sign_ns = api.nvim_create_namespace('')
```

Comment:
Can we consider making this non-anonymous?

Reply:
#26568